errortypes.go

```diff
@@ -116,7 +116,7 @@ func WithType(err error, errType ConstError) error {
 // interface.
 func Timeoutf(format string, args ...interface{}) error {
 	return newLocationError(
```

Comment:
Based on the docs from `go tool vet` we should do this:
```go
func Timeoutf(format string, args ...interface{}) error {
	if false {
		_ = fmt.Sprintf(format, args...) // enable printf checking
	}
	return newLocationError(
		makeWrappedConstError(Timeout, format, args...),
		1,
	)
}
```

Reply:
We can do one better, and put this check inside `makeWrappedConstError`.

Reply:
only reason I put it at the exported level was to save the linter from recursing further than it needs.